docker/artemis/Dockerfile

```diff
@@ -83,7 +83,7 @@ FROM ${WAR_FILE_STAGE} as war_file
 #-----------------------------------------------------------------------------------------------------------------------
 # runtime stage
 #-----------------------------------------------------------------------------------------------------------------------
-FROM docker.io/library/eclipse-temurin:17-jdk as runtime
+FROM docker.io/library/eclipse-temurin:17-jre as runtime
```

Comment:
This breaks Java plagiarism checks: https://github.com/jplag/JPlag/issues/845

Reply:
I guess this will then not be a viable option to do this 😞 

Thanks for notifying me.